scripts/pr_compliance.py

```diff
@@ -402,7 +402,7 @@ def make_default_validator(body, debug=False):
         "### Documentation Update",
         {"_Describe any necessary documentation update if there is any new feature, config, or user-facing change_",
         "",
-        "- _The config description must be updated if new configs are added or the default value of the configs are changed_",
+        "- _The config description must be updated if new configs are added or the default value of the configs are changed. If not, put \"N/A\"._",
```

Comment:
Maybe put anything there, like `none` or `nothing`, or just like the other checks to give some options. What is expected for the answer, yes or no ?

Reply:
As long as the user provide a description, the validation of this section passes.

Reply:
Maybe we should use `none` to keep in line with the other checks.

Reply:
Makes sense.  Addressed.